lib/alexa.js

```diff
@@ -217,4 +222,4 @@ process.on('uncaughtException', function(err) {
 
 module.exports.LambdaHandler = alexaRequestHandler;
 module.exports.CreateStateHandler = createStateHandler;
-module.exports.StateString = _StateString;
\ No newline at end of file
+module.exports.StateString = _StateString;
```

Comment:
I believe this means there was no line ending and now there is. I'm assuming this is an acceptable change? There really should always be a newline character.